builder/qemu/step_configure_qmp.go

```diff
@@ -26,9 +26,10 @@ type stepConfigureQMP struct {
 }
 
 func (s *stepConfigureQMP) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
+	config := state.Get("config").(*Config)
 	ui := state.Get("ui").(packer.Ui)
 
-	if !s.VNCUsePassword {
```

Comment:
You've eliminated the use of VNCUsePassword altogether here. We should probably remove it from the step struct.

Reply:
Ah indeed. Good catch!

Reply:
Its now removed.